src/lib/core/style-utils/style-utils.ts

```diff
@@ -16,8 +16,8 @@ import {LAYOUT_CONFIG, LayoutConfigOptions} from '../tokens/library-config';
 @Injectable({providedIn: 'root'})
 export class StyleUtils {
 
-  constructor(@Optional() private _serverStylesheet: StylesheetMap,
```

Comment:
Can you explain why all of the instances of `@Optional()` were removed? Due to TypeScript? Ivy?